easylist/easylist_adservers.txt

```diff
@@ -4441,6 +4446,7 @@
 ||seiya.work^$third-party
 ||sekindo.com^$third-party
 ||selectablemedia.com^$third-party
+||selfpua.com^$third-party
```

Comment:
Hi @smed79 

Have tested this domain, and can't get it to resolve with `selfpua.com` or `www.selfpua.com`. Do you have another url, from which you have chosen to block this at  second level?

Reply:
Hi,

==> `https://static.selfpua.com/mnpw.js`

Reply:
Thanks :+1: added it to own source :)